packages/frontity/src/cli/unknown.ts

```diff
@@ -12,7 +12,9 @@ const unknown = (command: string, program: CommanderStatic) => {
   console.log(chalk.red(`Unknown command: ${chalk.bold(command)}`));
 
   const availableCommands: string[] = program.commands.map((c) => c._name);
-  const suggestion: string | string[] = didYouMean(command, availableCommands);
+  const suggestion: string = availableCommands.find(
+    (c) => leven(c, command) < c.length * 0.4
```

Comment:
OK, that's a better threshold, @jamesgeorge007 . :+1:

Now, I think we should return the one with the least distance under that threshold, something like this:

```ts
const [suggestion] = availableCommands
  // Compute the distance of each command and return it along with the name.
  .map((name) => ({ name, distance: leven(name, command) }))
  // Filter commands by threshold.
  .filter(({ name, distance }) => distance < name.length * 0.4)
  // Sort them so the one with the least distance would be the first one.
  .sort((c1, c2) => c1.distance - c2.distance)
  // Map them and return only the command name.
  .map((c) => c.name)
```

That way we would be destructuring the first element of the available commands, filtered by the threshold and sorted by distance.

BTW, you can go to any of the example projects in the monorepo - for example, `/examples/mars-theme-example/` -and test the `npx frontity` commands there. :slightly_smiling_face: